### PR TITLE
Added additional information to search term page

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added additional information to search term page

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/search-terms-add.html

### CODE REVIEW

File: src/marketing/search-terms-add.md [hunk 1]
```diff
@@ -36,8 +36,34 @@ _Search Terms_
 
 ## Delete a Search Term
```
Comment (leslietilling) [inline src/marketing/search-terms-add.md:37]: We should probably fix capitalization in other parts of page too. "search term" is not really a proper noun and subheadings should be sentence case.

```suggestion
## Delete a search term
```

File: src/marketing/search-terms-add.md [hunk 2]
```diff
@@ -36,8 +36,34 @@ _Search Terms_
...
 
+There are two methods for deleting a Search Term -- from the grid and on the edit page.
```
Comment (leslietilling) [inline src/marketing/search-terms-add.md:39]:
```suggestion
There are two methods for deleting a search term -- from the grid and on the edit page.
```

File: src/marketing/search-terms-add.md [hunk 3]
```diff
@@ -36,8 +36,34 @@ _Search Terms_
...
+
+**Method 1:** In the Search Terms grid.
```
Comment (leslietilling) [inline src/marketing/search-terms-add.md:41]:
```suggestion
**Method 1:** In the _Search Terms_ grid
```

File: src/marketing/search-terms-add.md [hunk 4]
```diff
@@ -36,8 +36,34 @@ _Search Terms_
...
+
 1. In the list, select the checkbox of the term to be deleted.
 
 1. In the upper-left corner of the list, set **Actions** to `Delete`.
 
 1. When complete, click <span class="btn">Submit</span>.
+
+**Method 2:** On the Edit a Search Term page.
```
Comment (leslietilling) [inline src/marketing/search-terms-add.md:49]:
```suggestion
**Method 2:** On the _Edit a Search Term_ page
```

File: src/marketing/search-terms-add.md [hunk 5]
```diff
@@ -36,8 +36,34 @@ _Search Terms_
...
+
+1. On the _Admin_ sidebar, go to **Marketing** > _SEO & Search_ > **Search Terms**.
+
+1. Find the search term to be deleted and open it in edit mode.
+
+1. Click <span class="btn">Delete Search</span> button.
```
Comment (leslietilling) [inline src/marketing/search-terms-add.md:55]:
```suggestion
1. Click <span class="btn">Delete Search</span>.
```

File: src/marketing/search-terms-add.md [hunk 6]
```diff
@@ -36,8 +36,34 @@ _Search Terms_
...
+
+1. To confirm the action, click <span class="btn">OK</span>.
+
+## Grid column description
```
Comment (leslietilling) [inline src/marketing/search-terms-add.md:59]:
```suggestion
## Grid columns
```

File: src/marketing/search-terms-add.md [hunk 7]
```diff
@@ -36,8 +36,34 @@ _Search Terms_
...
+
+|Column|Description|
+|--- |--- |
+|Search Query|The query used to perform the search.|
+|Store|The store where the search query was applied.|
+|Results|Number of results found by query.|
+|Uses|Number of uses.|
+|Redirect URL|URL of the target page where user was redirected after performing the search.|
+|Suggested Terms|Determines if the query displays in suggested terms.|
+|Actions|Opens the product in Edit mode.|
```
Comment (leslietilling) [inline src/marketing/search-terms-add.md:69]:
```suggestion
|Actions|Opens the product in edit mode.|
```

File: src/marketing/search-terms-add.md [hunk 8]
```diff
@@ -36,8 +36,34 @@ _Search Terms_
...
+## Grid column description
+
+|Column|Description|
+|--- |--- |
+|Search Query|The query used to perform the search.|
+|Store|The store where the search query was applied.|
+|Results|Number of results found by query.|
+|Uses|Number of uses.|
+|Redirect URL|URL of the target page where user was redirected after performing the search.|
+|Suggested Terms|Determines if the query displays in suggested terms.|
```
Comment (leslietilling) [inline src/marketing/search-terms-add.md:68]:
```suggestion
|Suggested Terms|Determines if the query result displays suggested terms.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**